firstpartydata/tests/extractfpdforbidders/global-fpd-defined-site-content-data.json

```diff
@@ -75,9 +75,6 @@
           "id": "1"
         },
         "content": {
-          "len": 900,
-          "title": "reqContentTitle",
-          "season": "reqContentSeason",
           "data": [
```

Comment:
@mansinahar @bsardo ok, I got it now. This data appears in resolved FPD if `if fpdConfigSite != nil {` which is not the case for this test. This test covers this scenario (bidder specific fpd site.content takes precedence over global site.content.data):
```
if !bidderFpdSiteContentPresent && openRtbGlobalFPD != nil && len(openRtbGlobalFPD[siteContentDataKey]) > 0 {
		//bidder specific fpd site.content takes precedence over global site.content.data
---->.      newSite.Content = &openrtb2.Content{Data: openRtbGlobalFPD[siteContentDataKey]}
	}
```

Reply:
Basically Bidder is defined in global fpd list, but doesn't have fpd config.